src/libraries/Directory.Build.targets

```diff
@@ -219,13 +219,7 @@
   <Target Name="GetDocumentationFile"
           Returns="$(DocumentationFile)"/>
 
-  <Choose>
-    <When Condition="'$(IsPrerelease)' != 'false' AND '$(TargetFramework)' != '$(NetCoreAppCurrent)'">
-      <PropertyGroup>
-        <DefineConstants>$(DefineConstants),USE_INTERNAL_ACCESSIBILITY</DefineConstants>
-      </PropertyGroup>
-    </When>
-  </Choose>
+  <DefineConstants Condition="'$(IsPrerelease)' != 'false' AND '$(TargetFramework)' != '$(NetCoreAppCurrent)'">$(DefineConstants),USE_INTERNAL_ACCESSIBILITY</DefineConstants>
```

Comment:
This has to be inside a PropertyGroup, that's why I suggested doing it in:

https://github.com/dotnet/runtime/blob/master/src/libraries/Directory.Build.targets#L15